website/docs/r/video_analyzer.html.markdown

```diff
@@ -0,0 +1,132 @@
+description: |-
+  Manages a Video Analyzer.
+---
+
+# azurerm_video_analyzer
+
+Manages a Video Analyzer.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "video-analyzer-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_storage_account" "example" {
+  name                     = "examplestoracc"
+  resource_group_name      = azurerm_resource_group.example.name
+  location                 = azurerm_resource_group.example.location
+  account_tier             = "Standard"
+  account_replication_type = "GRS"
+}
+
+resource "azurerm_user_assigned_identity" "example" {
+  name                = "exampleidentity"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_role_assignment" "contributor" {
+  scope                = azurerm_storage_account.example.id
+  role_definition_name = "Storage Blob Data Contributor"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_role_assignment" "reader" {
+  scope                = azurerm_storage_account.example.id
+  role_definition_name = "Reader"
+  principal_id         = azurerm_user_assigned_identity.example.principal_id
+}
+
+resource "azurerm_video_analyzer" "example" {
+  name                = "exampleanalyzer"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  storage_account {
+    id          = azurerm_storage_account.example.id
+    identity_id = azurerm_user_assigned_identity.example.id
+  }
+
+  identity {
+    type = "UserAssigned"
+    identity_ids = [
+      azurerm_user_assigned_identity.example.id
+    ]
+  }
+
+  tags = {
+    environment = "staging"
+  }
+
+  depends_on = [
+    azurerm_user_assigned_identity.example,
+    azurerm_role_assignment.contributor,
+    azurerm_role_assignment.reader,
+  ]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Specifies the name of the Video Analyzer. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Video Analyzer. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists. Changing this forces a new resource to be created.
+
+* `storage_account` - (Required) An `storage_account` blocks is defined below.
+
+* `identity` - (Required) An `identity` block is defined below.
+
+* `tags` - (Optional) A mapping of tags assigned to the resource.
+
+---
+
+A `storage_account` block supports the following:
+
+* `id` - (Required) Specifies the ID of the Storage Account that will be associated with the Video Analyzer instance.
+
+* `identity_id` - (Required) ID for User Assigned Managed Identity resources to be assigned.
```

Comment:
perhaps this'd be clearer as:

```suggestion
* `user_assigned_identity_id` - (Required) Specifies the User Assigned Identity ID which should be assigned to a access this Storage Account.
```

Reply:
This again, comes down to, do we want to make this change if other resources don't use this term?

Reply:
As per the other thread, we may as well use `user_assigned_identity_id` as that will converge with other resources when we are able to update them